src/vs/workbench/browser/parts/views/viewsService.ts

```diff
@@ -143,7 +143,7 @@ export class ViewsService extends Disposable implements IViewsService {
 		for (const viewDescriptor of views) {
 			const disposables = new DisposableStore();
 			disposables.add(this.registerOpenViewAction(viewDescriptor));
-			disposables.add(this.registerFocusViewAction(viewDescriptor, composite?.name ?? CATEGORIES.View));
+			disposables.add(this.registerFocusViewAction(viewDescriptor, composite?.name && composite.name !== composite.id ? composite.name : CATEGORIES.View));
```

Comment:
Is the parameter after `viewDescriptor` used to put in the `View: ` part?

Reply:
Yes, the second parameter is the category for the command palette.  Our logic here is: if the composite name is the same as its id, its been generated and is an ugly string

Reply:
@sbatten  Thanks for fixing it. Is it better if we set the title of the generated container to be **Views** so that we will not run into such issues again?

https://github.com/microsoft/vscode/blob/e99ec0c6e5483aa593745933a190903cbea060b6/src/vs/workbench/services/views/browser/viewDescriptorService.ts#L466-L466